doc/proxy-metrics.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "Proxy Metrics"
+docpage = true
+[menu.docs]
+  parent = "proxy-metrics"
++++
+
+The Conduit proxy exposes metrics that describe the traffic flowing through the
+proxy.  These metrics are exposed in Prometheus format on the proxy control port
+(4190 by default) at the `/metrics` path.  The following metrics will be
```

Comment:
In [recent discussions](https://gitter.im/runconduit/proxy?at=5aa18dff6fba1a703ae3fc9f) I believe we determined that we want to expose the metrics in plaintext HTTP/1 on a separate server (and thus on its' own port).

Reply:
Cool.  Until this is implemented, this is kind of a moving target.  Would you mind if we just update this with the correct default port in the same PR as the new listener is added to the proxy?